core/src/main/java/com/sequenceiq/cloudbreak/service/blueprint/BlueprintService.java

```diff
@@ -129,27 +123,14 @@ public Blueprint create(Workspace workspace, Blueprint blueprint, Collection<Map
         return getAllAvailableInWorkspace(workspace);
     }
 
-    public Set<BlueprintView> getAllAvailableViewInWorkspace(Long workspaceId) {
+    public Set<Blueprint> getAllAvailableViewInWorkspace(Long workspaceId) {
```

Comment:
What's the reason behind reverting to Blueprint instead of BlueprintView? I checked the git logs and I see that the BlueprintView was added because of performance issues?